src/core/Table/Table.test.tsx

```diff
@@ -2445,3 +2445,189 @@ it('should not have `draggable` attribute on columns with `disableReordering` en
   expect(headerCells[3].getAttribute('draggable')).toBe('true'); // Description column
   expect(headerCells[4].getAttribute('draggable')).toBeFalsy(); // View column
 });
+
+it('should render empty action column with column manager', () => {
+  const columns: Column<TestDataType>[] = [
+    {
+      Header: 'Header name',
+      columns: [
+        {
+          id: 'name',
+          Header: 'Name',
+          accessor: 'name',
+        },
+        {
+          id: 'description',
+          Header: 'Description',
+          accessor: 'description',
+        },
+        {
+          id: 'view',
+          Header: 'View',
+          Cell: () => 'View',
+        },
+        ActionColumn({ hasColumnManager: true }),
+      ],
+    },
+  ];
+  const { container } = renderComponent({
+    columns,
+    data: mockedData(),
+    isSelectable: true,
+    subComponent: (row) => (
+      <div>{`Expanded component, name: ${row.original.name}`}</div>
+    ),
+  });
+
+  const columnManager = container.querySelector('.iui-button') as HTMLElement;
```

Comment:
Maybe search for the button in the Table header in the last cell. Because it looks like just any random button.

Reply:
Changed